google-cloud-speech/lib/google/cloud/speech/version.rb

```diff
@@ -16,7 +16,7 @@
 module Google
   module Cloud
     module Speech
-      VERSION = "0.27.0"
+      VERSION = "0.27.1"
```

Comment:
Can you revert the change to the `VERSION`? We change this during the release process, as [documented here](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/RELEASING.md).